source/manual/alerts/RouterErrorRatioTooHigh.html.md

```diff
@@ -0,0 +1,35 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Router error ratio too high
+parent: "/manual.html"
+layout: manual_layout
+section: Pagerduty alerts
+---
+
+You can find the router request error rates on this dashboard:
+
+- [5xx Router Requests][router-5x https://grafana.eks.production.govuk.digital/d/uBEWkijnz/router-request-rates-errors-durations?orgId=1 x-request-rates-grafana]
+
+You can also view the 500+503 error rates across all applications on this dashboard:
+
+- [5xx app Requests][app-5xx-error-rates-grafana]
+
+## description
```

Comment:
optional: I actually love these lowercase titles (they seem comfortingly retro to me, like [DEC branding](https://upload.wikimedia.org/wikipedia/commons/f/f3/DEC_System_10_Users_Handbook_%2815272018349%29.jpg) from the 70s-80's 😀) though I suspect [style guide says no](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#capitalisation) :( (your call though!)

Reply:
going to leave it 👍